### PR TITLE
Ugly game implementation

### PR DESCRIPTION
**Pull request by Ranray**
Ugly implementation of the game, with  synchronize issues.

### CODE REVIEW

File: Soccar [UI]/src/nl/socnet/message/MovePlayerMessage.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ */
+@MessageEvent(id = MessageConstants.MOVE_PLAYER_MESSAGE_ID, handler = MovePlayerMessageHandler.class)
+public final class MovePlayerMessage extends Message {
+
+    private final String username;
+    private final SteerAction steerAction;
+    private final HandbrakeAction handbrakeAction;
+    private final ThrottleAction throttleAction;
+
+    public MovePlayerMessage(String username, SteerAction steerAction, HandbrakeAction handbrakeAction, ThrottleAction throttleAction) {
```
Comment (ghost) [inline Soccar [UI]/src/nl/socnet/message/MovePlayerMessage.java:23]: MINOR Document this public constructor. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [UI]/src/nl/soccar/ui/fx/drawable/CarUiFx.java [hunk 2]
```diff
@@ -54,10 +59,21 @@ public CarUiFx(GameCanvas canvas, Car car, CarPhysics physics, TeamColour colour
...
+        if (ClientController.getInstance().getCurrentPlayer().getUsername().equals(super.getModel().getPlayer().getUsername())) {
+            SteerAction steerAction = Keyboard.getSteerAction();
+            HandbrakeAction handbrakeAction = Keyboard.getHandbrakeAction();
+            ThrottleAction throttleAction = Keyboard.getThrottleAction();
+
+            super.getModel().setSteerAction(steerAction);
+            super.getModel().setHandbrakeAction(handbrakeAction);
+            super.getModel().setThrottleAction(throttleAction);
+
+            // TODO : Change location of sending this message (Ugly implementation for now)
```
Comment (ghost) [inline Soccar [UI]/src/nl/soccar/ui/fx/drawable/CarUiFx.java:72]: INFO Complete the task associated to this TODO comment. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1135)

File: Soccar [UI]/src/nl/socnet/message/PlayerMovedMessage.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.PLAYER_MOVED_MESSAGE_ID, handler = PlayerMovedMessageHandler.class)
+public final class PlayerMovedMessage extends Message {
+
+    private final SteerAction steerAction;
+    private final HandbrakeAction handbrakeAction;
+    private final ThrottleAction throttleAction;
+    
+    public PlayerMovedMessage(SteerAction steerAction, HandbrakeAction handbrakeAction, ThrottleAction throttleAction) {
```
Comment (ghost) [inline Soccar [UI]/src/nl/socnet/message/PlayerMovedMessage.java:22]: MINOR Document this public constructor. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by ghost** [comment]
SonarQube analysis reported 3 issues
* MINOR 2 minor
* INFO 1 info

Watch the comments in this conversation to review them.

**Review by octotentakel — APPROVED** [review]

**Pull request merged**